README.md

```diff
@@ -27,14 +27,18 @@ data(package = 'DSMflow')
 
 Output from the CALSIM II operational model is used to generate hydrologic inputs for the CVPIA Decision Support Model (DSM). CALSIM II is the model used to simulate California State Water Project (SWP) / Central Valley Project (CVP) operations. CALSIM II was developed in the Water Resource Integrated Modeling System (WRIMS model engine or WRIMS), a generalized water resources modeling system for evaluating operational alternatives of large, complex river basins.
 
-The 2017 DSM uses the existing conditions simulation from the 2009 CalSim-II Draft SWP Delivery Reliability Studies 2009. The CALSIM II run is a Reclamation product used to replicate current operations for comparison with proposed adjustments under an ongoing Endangered Species Act consultation with the National Marine Fisheries Service.
+The 2017 DSM can use the existing conditions simulation from the 2009 CalSim-II Draft SWP Delivery Reliability Studies 2009. The CALSIM II run is a Reclamation product used to replicate current operations for comparison with proposed adjustments under an ongoing Endangered Species Act consultation with the National Marine Fisheries Service.
```

Comment:
Instead of staying the DSM can use say what the different options are you could describe that there are 2 versions and then list what calsim data each version uses